build/package/Microsoft.DotNet.Cli.Installer.DEB.props

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- dotnet deb-tool -->
+  <PropertyGroup>
+    <DotnetDebToolPackageId>dotnet-deb-tool</DotnetDebToolPackageId>  
+    <DotnetDebToolVersion Condition=" '$(DotnetDebToolVersion)' == '' ">1.0.0-*</DotnetDebToolVersion>
+    <DotnetDebToolConsumerProjectName>dotnet-deb-tool-consumer</DotnetDebToolConsumerProjectName>
+    <DotnetDebToolDir>$(IntermediateDirectory)/$(DotnetDebToolConsumerProjectName)</DotnetDebToolDir>
+    <DotnetDebToolPackageSource Condition=" '$(DotnetDebToolPackageSource)' == '' "></DotnetDebToolPackageSource>
+  </PropertyGroup>  
+
+  <ItemGroup>
+    <DotnetDebToolProjectJsonLines Include="{" />
+    <DotnetDebToolProjectJsonLines Include="%20%20%20%20&quot;version&quot;: &quot;1.0.0-*&quot;," />
```

Comment:
This isn't really readable.  How about we put this in a .json file and then read that file here?